dimod/reference/composites/fixedpolyvariable.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+"""
+A composite that fixes the variables provided and removes them from the binary
+polynomial model before sending to its child sampler.
+"""
+
+from dimod.core.polysampler import ComposedPolySampler
+from dimod.higherorder.polynomial import BinaryPolynomial
+from collections import defaultdict
+import dimod
+
+
+__all__ = ['FixedPolyVariableComposite']
+
+
+class FixedPolyVariableComposite(ComposedPolySampler):
+    """Composite to fix variables of a problem to provided.
+
+    Fixes variables of a BinaryPolynomial and modifies linear and k-local terms
```

Comment:
in this sentence it should be `binary polynomial` (the noun) rather than `BinaryPolynomial` (the object).